packages/oui-datagrid/README.md

```diff
@@ -428,6 +428,82 @@ You can use `row-loader`. It take the current row as argument and must return a
 </oui-datagrid>
 ```
 
+### Refresh
+
+If you want to refresh the datagrid, you will have to use a service.
```

Comment:
I would avoid too much conditional, maybe `To refresh a datagrid, you need to use the angular datagrid service` or `The data inside a datagrid can be modified after his initialisation with the angular datagrid service` could do the work

Reply:
Hum, yes the doc could be better...

Reply:
Like that.